bin/bin.js

```diff
@@ -8,7 +8,7 @@ const Fixer = require(`../lib/fixer`);
 
 program
   .version(pkg.version)
-  .usage(`<directory> [options]`)
+  .usage(`<directories or files> [options]`)
```

Comment:
Commander writes something like this in their examples `<path ...>` to express many

Reply:
I changed it to `<paths ...>`